CMakeLists.txt

```diff
@@ -110,11 +108,8 @@ if (BUILD_RUNTIME)
   add_subdirectory(${CMAKE_HOME_DIRECTORY}/_studio)
 endif()
 
-if(RUN_TESTS)
-    enable_testing()
-endif()
-
-if (BUILD_TESTS AND GTEST_FOUND)
+if (BUILD_TESTS)
```

Comment:
That's correct. Do _not_ add GTEST_FOUND here. If someone configures build with BUILD_TESTS=ON he should get configuration error  much earlier (from FindGTest.cmake at the moment). Maybe that's not exactly so at the moment, but that's where we are heading to.